data/features.yml

```diff
@@ -371,6 +443,8 @@ browser_extension:
   product_team: repo_management
   maturity: ga
   documentation_link: https://docs.sourcegraph.com/integration/browser_extension
```

Comment:
We don't own that, does the Growth team maybe?

Reply:
Ok, thanks!